generators/server/templates/dotnetcore/src/Project/Controllers/UserController.cs.ejs

```diff
@@ -87,8 +87,9 @@ namespace <%= namespace %>.Controllers {
         public ActionResult<IEnumerable<User>> GetAllUsers(IPageable pageable)
         {
             _log.LogDebug("REST request to get a page of Users");
-            var page = _userManager.Users.UsePageable(pageable);
-            var headers = PaginationUtil.GeneratePaginationHttpHeaders(page, HttpContext.Request);
+            var sortExpressions = <%= namespace %>.Models.User.GetSortExpressions(pageable.Sort);
```

Comment:
Why use namespace here ?

Reply:
because compiler try to use ControllerBase.User class instead of our specific Models.User

Reply:
yes